src/containers/ConceptPage/ConceptForm/ConceptForm.jsx

```diff
@@ -223,84 +214,29 @@ class ConceptForm extends Component {
       createMessage,
       fetchConceptTags,
       fetchStateStatuses,
-      history,
       inModal,
       isNewlyCreated,
       licenses,
-      locale,
       onClose,
-      onUpdate,
       subjects,
       t,
       translateConcept,
-      ...rest
     } = this.props;
     const { savedToServer, translateOnContinue } = this.state;
-    const panels = ({ errors, touched }) => [
```

Comment:
Nysgjerrig på hvorfor har du gjort denne om fra en liste?
Er slik det er gjort det fleste steder. Har lest artikler som anbefaler å gjøre det slik for å kunne ha oversikt over innhold, raskt legge til/ fjerne innhold, og for å kunne skrive mindre jsx/"html".
Ser du har laget en AccordionSection, kunne du f.eks. iterert gjennom listen og opprettet en AccordionSection komponent for hver?

Reply:
* Første grunnen er fordi jeg hele tiden måtte scrolle opp og ned hundre linjer for å se på den samme contexten(Panels)
Den samme informasjonen som er i lista panels ligger nå i AccordionSection, som samler alle propsene på et sted sammen med htmlen den skal brukes i.

* Det er mer eksplisitt. Her er det en accordionSection med id x, tittel y, og disse klassene. Du kan se alt du trenger å vite på en skjerm.

Når det gjelder punktene dine så syntes jeg det er mer oversiktlig å håndtere en context på et sted, det er bare å legge til og fjerne ting på akkurat samme måten som i lista, bare med en annen struktur på informasjonen, Og jeg er ganske sikker på at jeg endte opp med færre linjer på denne måten.

Kunne selvsagt iterert gjennom en liste og matet det inn i AccordionSections, men da mister vi igjen det jeg syntes var positivt med å håndtere den samme contexten sammen.

Dette er selvsagt bare min mening, så om du og andre er uenig så er det bare å slå meg ned. Det var derfor jeg ønsket feedback, da jeg forstår at om man skal endre det slik så burde man gjøre det overalt, og jeg har ikke lyst til å legge det på dere mot deres vilje.

Reply:
https://github.com/NDLANO/editorial-frontend/pull/919/commits/b8d444f3c06ce22f7a0e8e69e8184857d9192336
Her er en diff for den endringen alene